README.md

```diff
@@ -1,189 +1,157 @@
-					}
-				}
-			],
-            "alg": "es256",
-            "private_key": "es256_private.key",
-            "sign_cert": "es256_certs.pem",
-            "ta_url": "http://timestamp.digicert.com"
-		}
-    ],
-	"required": [
-		"assertions",
-	],
-	"properties": {
-		"vendor": {
-			"type": "string",
-			"description": "Typically an Internet domain name (without the TLD) for the vendor (i.e. `adobe`, `nytimes`)"
-		},
-		"claim_generator": {
-			"type": "string",
-			"description": "A UserAgent string that will let a user know what software/hardware/system produced this Manifest - names should not contain spaces (defaults to c2patool)"
-		},
-		"title": {
-			"type": "string",
-			"description": "A human-readable string to be displayed as the tile for this Manifest (defaults to embedded file name)"
-		},
-		"credentials": {
-			"type": "object",
-			"description": "An array of W3C verifiable credentials objects defined in the c2pa assertion specification. Section 7"
-		},
-		"parent": {
-			"type": "string",
-			"format": "Local file system path",
-			"description": "A file path to the source image that was modified by this Manifest (if any)"
-		},
-        "Ingredients": {
-			"type": "array of string",
-			"format": "Array of local file system paths",
-			"description": "File paths to images that were used to modify the image referenced by this Manifest (if any)"
-		},
-		"assertions": {
-			"type": "object",
-			"description": "Objects with label, and data - standard c2pa labels must match values as defined in the c2pa assertion specification"
-		},
-		"alg": {
-			"type": "string",
-			"format": "Local file system path",
-			"description": "Signing algorithm: one of [ ps256 | ps384 | ps512 | es256 | es384 | es512 | ed25519]"
-		},
-		"ta_url": {
-			"type": "string",
-			"format": "http URL",
-			"description": "A URL to an RFC3161 compliant Time Stamp Authority"
-		},
-		"private_key": {
-			"type": "string",
-			"format": "Local file system path",
-			"description": "File path to a private key file"
-		},
-		"sign_cert": {
-			"type": "string",
-			"format": "Local file system path",
-			"description": "File path to signing cert file"
-		},
-		"base_path": {
-			"type": "string",
-			"format": "Local file system path",
-			"description": "File path to a folder to use as the base for relative paths in config"
-		},
-	},
-	"additionalProperties": false
-}
+**Note:** You may have need to update your `openssl` version if the above command does not work. You will likely need version 3.0 or later. You can check the version that is installed by typing `openssl version`.
+
+c2patool can also timestamp the embedded signature data.  This is useful for validating an asset when the embedded certificates have expired. If the config has a `ta_url` set, c2patool will attempt to timestamp the signature using the TA service at the provided URL. The TA must be [RFC3161](https://datatracker.ietf.org/doc/html/rfc3161) compliant. For example,
```

Comment:
The timestamp feature is independent from creating custom certs, we should move this out of the appendix, Note that the "config" referred to here is the same file referred to as manifest definition elsewhere. The name for the file should be aligned.